ui/cypress/e2e/collectors.test.ts

```diff
@@ -369,5 +369,31 @@ describe('Collectors', () => {
           })
       })
     })
+    describe('Label creation and searching', () => {
+      beforeEach(() => {
+        const description = 'Config Description'
+        cy.get('@org').then(({id}: Organization) => {
+          cy.createTelegraf('newteleg', description, id, 'zoebucket')
+        })
+        cy.reload()
```

Comment:
can you add one of those `cy.get({timeout})` lines here after the reload to ensure that the page has loaded?